Index: app/src/main/java/sv/edu/ues/fia/eisi/pedidoscafeteria/NuevaClase.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/sv/edu/ues/fia/eisi/pedidoscafeteria/NuevaClase.java	(date 1591056056225)
+++ app/src/main/java/sv/edu/ues/fia/eisi/pedidoscafeteria/NuevaClase.java	(date 1591056056225)
@@ -0,0 +1,5 @@
+package sv.edu.ues.fia.eisi.pedidoscafeteria;
+
+public class NuevaClase {
+}
+//hola jajaja
\ No newline at end of file
